### DISCUSSION TITLE
how to access old ui?

### DISCUSSION THREAD

**Discussion by kathrynmont** [aclap-dev/video-downloadhelper#1411] [Help and Q&A]
### Your question

Hi -- I'm a really old user who really liked the control I had for choosing a download site with the old UI.  Does anyone know if I can access it -
- not use the new sidebar thingy?  Thanks ... :)
Version 9.1.0.44 (stable), target mozilla, locale en-US.
Companion App installed (v2.0.19) Recheck 
License is verified/registered
C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe 

### Addon & CoApp version and details

Version 9.1.0.44 (stable), target mozilla, locale en-US.
Companion App installed (v2.0.19) Recheck 
License is verified/registered
C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by jcfrog** [comment]
Hi,
This should help: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#legacy-ui

**Reply by kathrynmont** [reply]
It worked! Wow ... just amaaazing that you took your time to so kindly answer my question -- thank you SO much! :))
